CI is broken on edna-site ...

```_____________________________________________ ERROR at setup of test_external_dataset_info _____________________________________________

tmpdir_factory = TempdirFactory(_tmppath_factory=TempPathFactory(_given_basetemp=None, _trace=<pluggy._tracing.TagTracerSub object at 0x7fd318ee36d8>, _basetemp=PosixPath('/tmp/pytest-of-tester/pytest-855')))

    @pytest.fixture(scope="module")
    def hdf5_with_external_data(tmpdir_factory):
        tmpdir = tmpdir_factory.mktemp("hdf5_with_external_data")
        master = str(tmpdir / "master.h5")
        external_h5 = str(tmpdir / "external.h5")
        external_raw = str(tmpdir / "external.raw")
    
        data = numpy.array([100, 1000, 10000], numpy.uint16)
        tshape = (1,) + data.shape
    
        with h5py.File(master, "w") as fmaster:
            dset = fmaster.create_dataset("data", data=data)
    
            fmaster["int"] = h5py.SoftLink("data")
    
>           layout = h5py.VirtualLayout(shape=tshape, dtype=data.dtype)
E           AttributeError: module 'h5py' has no attribute 'VirtualLayout'

build/lib.linux-x86_64-3.7/silx/io/test/test_h5link_utils.py:24: AttributeError
```

--- a/src/silx/io/test/test_h5link_utils.py
+++ b/src/silx/io/test/test_h5link_utils.py
@@ -7,7 +7,6 @@ from silx.io import h5link_utils
 
 
 @pytest.fixture(scope="module")
-@pytest.mark.skipif("VirtualSource" not in dir(h5py), reason="h5py is too old")
 def hdf5_with_external_data(tmpdir_factory):
     tmpdir = tmpdir_factory.mktemp("hdf5_with_external_data")
     master = str(tmpdir / "master.h5")
@@ -95,6 +94,7 @@ def hdf5_with_external_data(tmpdir_factory):
     return tmpdir
 
 
+@pytest.mark.skipif("VirtualLayout" not in dir(h5py), reason="h5py is too old")
 def test_external_dataset_info(hdf5_with_external_data):
     tmpdir = hdf5_with_external_data
     master = str(tmpdir / "master.h5")

